src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.body {
+  font-family: Roboto, serif;
```

Comment:
The font-family `Roboto` is used here, but according to the task requirements, it should be included using `@font-face`. Ensure that you define `Roboto` using `@font-face` if it's not already included.